src/components/Autocomplete/index.tsx

```diff
@@ -249,14 +232,9 @@ class AutocompleteInt<T> extends React.Component<AutocompleteProps<T> & FormCont
   }
 
   private show = () => {
-    const { onFocus } = this.props;
-
-    this.setState(
-      () => ({
-        open: true,
-      }),
-      onFocus,
-    );
+    this.setState(() => ({
+      open: true,
+    }));
```

Comment:
no need to use a factory. just `this.setState({ open: true })` looks better ;)

Reply:
Fixed.